src/components/NcSelect/NcSelect.vue

```diff
@@ -1182,8 +1184,10 @@ body {
 }
 
 .vs__dropdown-menu {
-	border-color: var(--color-primary-element) !important;
+	border-color: var(--color-main-text) !important;
+	outline: 2px solid var(--color-main-background);
```

Comment:
This outline adds a not so nice gap between the select toggle and the dropdown-menu. But I am not sure we can fix this.
![grafik](https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/a886644c-058a-4730-96a9-7c81e498ead9)

More obvious with adjusted outline color
![grafik](https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/ce3a2a0d-a02f-42b6-807e-ea348f6c1024)

@susnux Any idea here?

Reply:
well, without this `outline` it looks odd too like from your video:

https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/787efbcd-0d16-436f-950c-221f662c2c0e

I think it is better to keep `outline`. And all other improvements we can do after a11y certification

Reply:
> well, without this `outline` it looks odd too like from your video:

That's unfortunately true. One could fix it with an additional container around the dropdown and some CSS including a `margin-top: -2px;`, but that would require adjusting `@nextcloud/vue-select` which is out out scope here.

Reply:
I think the vertical position of the drop down needs to be changed. But not sure this is possible without changing  the vue-select

Reply:
> I think the vertical position of the drop down needs to be changed. But not sure this is possible without changing the vue-select

I tested this (manually). It won't help, since both elements, the trigger and the dropdown, have an outline.

Reply:
=(
could we merge this pr and improve left things in next  future?

Reply:
ok, i will merge it now, got enough approves